### PR TITLE
Kathrine Johnson JSON

### PR DESCRIPTION
**Pull request by bjordancodes**

### CODE REVIEW

File: People/KathrineJohnson.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "job": "Computer, Research Mathematician",
+        "notes": "From 1958 until her retirement in 1986, Johnson worked as an aerospace technologist, moving during her career to the Spacecraft Controls Branch. She calculated the trajectory for the May 5, 1961 space flight of Alan Shepard, the first American in space. She also calculated the launch window for his 1961 Mercury mission. She plotted backup navigation charts for astronauts in case of electronic failures. When NASA used electronic computers for the first time to calculate John Glenn's orbit around Earth, officials called on Johnson to verify the computer's numbers; Glenn had asked for her specifically and had refused to fly unless Johnson verified the calculations.",
+        "start_year": 1958,
+        "end_year": 1986
+    }],
+    "accomplishment": [{ 
+        "type": "Award", 
+        "description": "Kathrine received multiple individual honors and group awards for her outstanding work on NASA's Lunar Spacecraft and Operations", 
+        "short_description": "NASA Langley Research Center Special Achievement award", 
+        "year": [1971, 1980, 1985, 1986]
```
Comment (tthompson899) [inline People/KathrineJohnson.json]: I'm not too familiar if the array would work here...All other data for year is an integer. 
If you're sure it'll work for others pulling in the data - I'll merge...Please confirm.

File: People/KathrineJohnson.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    },
+    {
+        "type": "Presidential Medal of Freedom", 
+        "short_description": "President Barak Obama awarded Kathrine the Presidential Medal of Freedom", 
+        "year": 2015
+    },
+    {
+        "type": "Honorary Doctorate of Science", 
+        "short_description": "Katherine was awarded honorary doctorates from: Capitol College, laural, Maryland; Old Dominion University, Norfolk, Virginia; Spelman College; College of William and Mary, Williamsburg, Virigina.", 
+        "year": [2006, 2010, 2017, 2018]
```
Comment (tthompson899) [inline People/KathrineJohnson.json]: Same thing here...See above comment.

File: People/KathrineJohnson.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        "year": [1971, 1980, 1985, 1986]
+    },
+    {
+        "type": "Presidential Medal of Freedom", 
+        "short_description": "President Barak Obama awarded Kathrine the Presidential Medal of Freedom", 
+        "year": 2015
+    },
+    {
+        "type": "Honorary Doctorate of Science", 
+        "short_description": "Katherine was awarded honorary doctorates from: Capitol College, laural, Maryland; Old Dominion University, Norfolk, Virginia; Spelman College; College of William and Mary, Williamsburg, Virigina.",
```
Comment (tthompson899) [inline People/KathrineJohnson.json]: Also, these are from different schools so you could also just put another line for each of these with the year.

File: People/KathrineJohnson.json (hunk 1 above)
Comment (tthompson899) [reply People/KathrineJohnson.json]: You could also add to the description that it was for multiple years and add the years there but put the most recent year in the *year* attribute

**Comment by bjordancodes** [comment]
@tthompson899 updated! Sorry for the delay!

**Pull request merged**